content/ansible-tower-getting-started/2-cred/_index.md

```diff
@@ -177,3 +178,43 @@ The yellow output of the command indicates Ansible has actually done something (
 {{% notice tip %}}
 Try to click one of the output lines in the window showing the job output. A small window with a lot of information about the job like module args will open. Have a look and close it again.
 {{% /notice %}}
+
+## Execution Environments: A deeper look
+
+As promised let's look a bit deeper into execution environments. During the section covering ad hoc commands you have already seen you have to choose an execution environment, the same will hold true for running Playbooks later on. In your Automation Controller web UI, go to **Administration → Execution Environments**. You'll see a list of the configured execution environments and original location of the image, in our case the images are provided in the **quay.io** container registry. Here you could add your own registry with custom EE images, too.
```

Comment:
"During the section covering ad hoc commands you have already seen you have to choose an execution environment, the same will hold true for running Playbooks later on" - I find this sentence too long and confusing